docroot/themes/humsci/humsci_basic/src/scss/_base.scss

```diff
@@ -1,8 +1,18 @@
 @charset "UTF-8";
 
-// Decanter's use of global variables won't allow us to import with @use
-// @use 'decanter/core/src/scss/components/brand-bar/index';
-@import 'decanter/core/src/scss/decanter-no-markup';
+@import
+  'decanter/core/src/scss/decanter-no-markup',
+  'decanter/core/src/scss/core/normalize',
+  'decanter/core/src/scss/core/base',
+  'decanter/core/src/scss/core/helpers',
+  'decanter/core/src/scss/elements',
```

Comment:
this includes specific, overriding styles for h1-h6, lists, tables, and other certain elements. we can keep this for now, but might want to see how much we have to override to start building the colorful theme